Gemfile.jruby-2.3.lock.release

```diff
@@ -591,7 +591,7 @@ GEM
       simple_oauth (~> 0.3.0)
     tzinfo (1.2.4)
       thread_safe (~> 0.1)
-    tzinfo-data (1.2017.3)
```

Comment:
Does this contain the DB in use too ?  Should be cautious about this update ?

Reply:
Time zones do change with some regularity, I'm not too worried about it.